src/config.h

```diff
@@ -33,8 +33,10 @@ struct touchscreen_deadzone {
 };
 
 enum {
-  INPUT_SPECIAL_KEY_MODE = -10,
-  INPUT_SPECIAL_KEY_PAUSE
+  INPUT_SPECIAL_KEY_PAUSE = -0xffff,
```

Comment:
hum... `-config_code - 0xfff0`. i don't like these codes.
it make to hard to understand except deep think.(i think, it will return 0x1~0xf right?)

i don't know good solution,
but, just my opinion, i will choose like this instead that (but, it's still preference problem...);

```
#define KEYBOARD   0x00F00000 // 0x00F00000 ~ 0x00FFFFFF
#define CONTROLLER 0x0F000000 // 0x0F000000 ~ 0x0F0FFFFF
#define SPECIAL    0xF0000000 // 0xF0000000 ~ 0xF00FFFFF
```

then we don't need anymore negative or positive check
just need bitmask check

```
if (value & KEYBOARD) {
  kbd = value & ~KEYBOARD;
} else if (value & CONTROLLER) {
  ctrl = value & ~CONTROLLER;
} else if (value & SPECIAL) {
  special = value & ~SPECIAL;
}
```

so, we can define keys like this

```
enum {
  INPUT_KBD_F1          = 112 | KEYBOARD, // 0x00F00070
  INPUT_KBD_F2          = 113 | KEYBOARD, // 0x00F00071
  ...
  INPUT_BTN_LB          = 512  | CONTROLLER, // 0x0F000100
  INPUT_BTN_XBOX        = 1024 | CONTROLLER, // 0x0F000400
  ...
  INPUT_SPECIAL_KEY_LMB = 1 | SPECIAL, // 0xF0000001
  INPUT_SPECIAL_KEY_MMB = 2 | SPECIAL, // 0xF0000002
  ...
};
```